com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -287,6 +287,10 @@ internal static class ShaderPropertyId
 
     public struct PostProcessingData
     {
+        /// <summary>
+        /// Post process data used.
+        /// </summary>
+        public PostProcessData resources;
```

Comment:
Not ideal we have PostProcessData and PostProcessingData :( 
We should at least make it more clear what the data here is in docs, is it resources?

Reply:
Maybe lets make it internal for now.